src/scene/materials/standard-material-parameters.js

```diff
@@ -1,13 +1,18 @@
-function _textureParameter(name, channel = true) {
-    const tex = {};
-    tex[`${name}Map`] = 'texture';
-    if (channel)
-        tex[`${name}MapChannel`] = 'string';
-    tex[`${name}MapUv`] = 'number';
-    tex[`${name}MapTiling`] = 'vec2';
-    tex[`${name}MapOffset`] = 'vec2';
-    tex[`${name}MapRotation`] = 'number';
-    return tex;
+function _textureParameter(name, channel = true, vertexColor = true) {
```

Comment:
Still think we should change this name if we are going to introduce vertex colors. If we call it _modifierParameters and add texture as a boolean as well, this can be much more generic in my opinion.

Reply:
I can change the name, but what's the point of adding a boolean that will always be true?

Reply:
Can't we have only vertex color without texture? If so, then there is no need.

Reply:
Actually can we make this change in another PR?

`textureParameter` currently mirrors standard-material's `defineTex2D`, so if we change this naming we should also change that.

Since we're going to reorder members to be alphabetical as well, I'd rather we just do all that together in a separate PR.